mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -268,6 +294,10 @@ class ALSModel private[ml] (
 @Since("1.6.0")
 object ALSModel extends MLReadable[ALSModel] {
 
+  private val NaN = "nan"
+  private val Drop = "drop"
+  private[recommendation] final val supportedUnknownStrategies = Array(NaN, Drop)
```

Comment:
It would be ok, but I use `supportedUnknownStrategies` in the param comment for "supported values" too, so I think this might be best.